### DISCUSSION TITLE
[FTL] Could not create runner + CDN error

### DISCUSSION THREAD

**Discussion by rf-peixoto** [projectdiscovery/httpx#1544] [Q&A] [answered]
Hey, guys. All good? I'm having the following error when running nuclei v3.0.4, all templates are updated:

`[FTL] Could not create runner: Get "https://cdn.nuclei.sh": dial tcp: lookup cdn.nuclei.sh: no such host`

I researched some similar errors (https://github.com/projectdiscovery/httpx/issues/599). I'm not using any VPN, I don't seem to have any type of blocking on CloudFlare. For context, my script combines a number of recon tools (Link: https://github.com/rf-peixoto/Studies/blob/main/Code/Bash/izanami.sh Please ignore the parameter mixing and juggling to generate the output, this script is still under testing.). On line 65, my command is as follows:

`nuclei -env-vars -silent -l $1/subdomains.txt > $1/nuclei.txt`

The error interrupts the execution of the nuclei, causing the output file to be empty. In the environment variables I only have my shodan API. Also, I checked the `-validate` command and found only some templates with duplicate IDs. The problem is certainly my outdated environment (last update for Fedora 36), this is my test lab and it is no longer possible to update it without breaking all my projects. I intend to install a fresh system in the next few days, but I would like to know if there is another solution or if someone had the same problem. The command works perfectly on Kali 2023.3.

Thanks in advance!

**Comment by rf-peixoto** [comment]
Sorry, I commented on the wrong repo. -facepalm-

**Comment by ehsandeep** [comment, marked as answer]
@rf-peixoto this is possibly because you using outdated version of PD tool, httpx or nuclei or any other tool where cdn feature is used.

`https://cdn.nuclei.sh` is discontinued and cdn data loaded directly from code https://github.com/projectdiscovery/cdncheck, all the projects got uploaded to use latest code, so updating project to latest version should resolve the issue.
